### ISSUE TITLE
SettingsPanel conflict with World Quest Tracker

### ISSUE THREAD

**Issue by Aur0r4** [jaredbwasserman/drift#44]
Hello,

I think I've found a bug that seems to be related to my Drift configuration in combination with the AddOn World Quest Tracker:

I've tested this with a clean Interface and WTF folder, only Drift and World Quest Tracker are installed:

When you have changed position of 'SettingsPanel' AND World Quest Tracker is enabled, then you are not able to bind mouse-buttons like middle button to anything via Blizzard Keybinding.

When I remove this Config for SettingsPanel in SavedVariables ....

```Lua
	["SettingsPanel"] = {
		["relativeTo"] = "UIParent",
		["point"] = "LEFT",
		["relativePoint"] = "LEFT",
		["yOfs"] = 88.9012222290039,
		["xOfs"] = 466.0560607910156,
```

... keybinding is working fine.

This does not happen if you only change the scaling of the 'SettingsPanel'.

It took some time for me to figure this out, I hope it can help you. Not sure if it is something on Drifts end or World Quest Tracker or even Blizzard.

cheers
Aur0r4

**Comment by jaredbwasserman** [comment]
Hmm, that is a really interesting bug. Thank you for the thorough investigation! I'll take a look.

**Comment by jaredbwasserman** [comment]
I'm still looking into this. One (not great) workaround you can use in the meantime is keep middle (for example) clicking randomly on the button and eventually it will bind.

**Comment by jaredbwasserman** [comment]
I fixed this issue in Drift v2.2.0. There was some interesting interaction occurring with how Drift was implemented. This should be all fixed now, but please re-open if you're still seeing issues.
